src/game/scenes/arena.c

```diff
@@ -534,12 +537,26 @@ void arena_har_defeat_hook(int player_id, scene *scene) {
             player_winner->pilot->wins++;
             player_loser->pilot->losses++;
             if(player_id == 1) {
+                fight_stats->winnings = player_loser->pilot->winnings;
+                fight_stats->repair_cost = (1.0f - ((float)winner_har->health / (float)winner_har->health_max)) * 1000;
```

Comment:
Does this vary with difficulty?

Reply:
Eventually probably yes, but not in this implementation.